clients/bigquery/dialect/dialect.go

```diff
@@ -154,6 +154,15 @@ func (bd BigQueryDialect) BuildIsNotToastValueExpression(tableAlias constants.Ta
 	return fmt.Sprintf("COALESCE(%s != '%s', true)", colName, constants.ToastUnavailableValuePlaceholder)
 }
 
+func (bd BigQueryDialect) GetDedupeTableQuery(tableID sql.TableIdentifier, primaryKeys []string) string {
+	primaryKeysEscaped := sql.QuoteIdentifiers(primaryKeys, bd)
+	return fmt.Sprintf(`(SELECT * FROM %s QUALIFY ROW_NUMBER() OVER (PARTITION BY %s ORDER BY %s) = 1)`,
```

Comment:
Had to do this because BigQuery doesn't like DISTINCTs for JSON columns, see:

```
Column foo of type JSON cannot be used in SELECT DISTINCT
```

Reply:
Want to add a comment to this effect?

Reply:
Didn't see this. Yeah I'll add one